commcare_connect/templates/program/email/program_invite_notification.html

```diff
@@ -0,0 +1,12 @@
+<p>Hi,</p>
+
+<p>You have been invited to join the Program "{{ application.program.name }}".</p>
+
+<p>
+    <a href="{{ program_url }}">Click here</a> to review and apply for the Program.
+</p>
+
+<p>
+    Thank you,<br>
+    CommCare Connect
```

Comment:
This should just be Connect
```suggestion
    Connect
```